src/Pos/Ssc/GodTossing/Arbitrary.hs

```diff
@@ -43,7 +43,7 @@ import           Pos.Ssc.GodTossing.Types.Message (GtTag (..), MCCommitment (..)
 import           Pos.Ssc.GodTossing.Types.Types   (GtGlobalState (..),
                                                    GtSecretStorage (..))
 import           Pos.Ssc.GodTossing.VssCertData   (VssCertData (..))
-import           Pos.Types.Arbitrary.Core.Unsafe  ()
+import           Pos.Core.Types.Arbitrary.Unsafe  ()
 import           Pos.Util.Arbitrary               (Nonrepeating (..), makeSmall, sublistN,
                                                    unsafeMakePool)
 
```

Comment:
Now I realized that I am reviewing only subset of changes, maybe it's already moved. I will see it soon :)

Ignore if it's really moved.